presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -100,7 +100,8 @@ Constructors
 
 .. function:: ST_Point(x, y) -> Point
 
-    Returns a geometry type point object with the given coordinate values.
+    Returns a geometry type point object with the given longitude and latitude coordinate values. 
```

Comment:
We may need to be careful here not to create an impression that computations are done on a sphere. All computations are done on a 2d plane with (x, y) coordinates. One can treat a sphere as such..... with x = longiture and y = latitude... but they need to understand what they are doing... I think this is why we documented it as (x, y) and not (longitude, latitude).

Reply:
Good point! I could add your clarification with something like the following: 

"All computations of this function are done on a 2d plane - not a sphere- with (x, y) coordinates." 

Would that work? Do you have a better suggestion?

Reply:
I noticed we already have this:

```
Use ST_GeometryFromText and ST_GeomFromBinary functions to create geometry objects from WKT or WKB. In WKT/WKB, the coordinate order is (x, y). For spherical/geospatial uses, this implies (longitude, latitude) instead of (latitude, longitude).

The basis for the Geometry type is a plane. The shortest path between two points on the plane is a straight line. That means calculations on geometries (areas, distances, lengths, intersections, etc) can be calculated using cartesian mathematics and straight line vectors.
```

Do we need to add anything else?

Reply:
I saw that, and it helped me understand the context. Because the opener of the issue described that it was unclear to them whether to use (longitude, latitude) or (latitude, longitude) for ST_POINT, I wanted to revise the description of ST_POINT as that existing content didn't help them. 

How about this replacement of my current line 103, bringing your quoted material into context in the ST_POINT definition? 

"Returns a geometry type point object, using longitude and latitude coordinate values as ``x`` and ``y`` to calculate the point on a plane."

Reply:
> as that existing content didn't help them.

Is this the case? I don't think we even asked them if they read that paragraph, did we?

Reply:
The existing content you quote was present on the page when they opened the issue, but I agree that we didn't ask them if they read that paragraph. What would you suggest?

Reply:
I suggest asking them whether they read that part of the doc. If not, we need to remind ourselves to remember to read the full page and ask folks to do the same next time.

Reply:
or, at least search for "lat, long" on the page before assuming the doc is lacking.